liquibase-standard/src/main/java/liquibase/sqlgenerator/core/RenameColumnGenerator.java

```diff
@@ -80,6 +79,21 @@ public Sql[] generateSql(RenameColumnStatement statement, Database database, Sql
         };
     }
 
+    private String generateMysqlStatement(MySQLDatabase database, RenameColumnStatement statement) {
+        String sql;
+        try {
+            // https://dev.mysql.com/worklog/task/?id=10761 RENAME COLUMN introduced in 8.0
+            if (database.getDatabaseMajorVersion() >= 8) {
```

Comment:
@quangdutran I just remembered that `MariaDBDatabase` extends `MysqlDatabase`, so it will also get here and changes will be applied to `MariaDB >= 8` . We will need to change this validation to account for `MariaDB >= 10.5.2` to take advantage of this change too. I think you could remove the cast to `MysqlDatabase` when invoking the method and change the validation to something like
```
boolean isMariaDB = database instanceof MariaDBDatabase;
boolean isNewerVersion = isMariaDB
    ? (database.getDatabaseMajorVersion() == 10 && database.getDatabaseMinorVersion() >= 5)
        || database.getDatabaseMajorVersion() >= 11
    : database.getDatabaseMajorVersion() >= 8;

if (isNewerVersion) {

```